demos/fleet_provisioning/fleet_provisioning_with_csr/demo_config.h

```diff
@@ -91,18 +91,26 @@
 
 /**
  * @brief Path the file containing the provisioning claim certificate. This
- * certificate is used to authenticate with AWS IoT for provisioning by claim.
+ * certificate is used to connect to AWS IoT Core and use Fleet Provisioning
+ * APIs to provision the client device. This is used for the "Provisioning by
+ * Claim" provisioning workflow.
  *
  * For information about provisioning by claim, see the following AWS documentation:
  * https://docs.aws.amazon.com/iot/latest/developerguide/provision-wo-cert.html#claim-based
  *
- * @note This certificate should be PEM-encoded.
+ * @note This certificate should be PEM-encoded. The certificate should be
+ * registered on AWS IoT Core beforehand. It should have an AWS IoT policy to
+ * allow it to access only the Fleet Provisioning APIs. An example policy for
+ * the claim certificates for this demo is available in the claim_policy.json
+ * file in the demo directory. In the example, replace us-west-2 with your AWS
```

Comment:
Same comment here about not providing actual account information in the source code!